ui.apps/src/main/content/jcr_root/apps/venia/components/commerce/productlist/v1/productlist/clientlibs/dist/js/navhelper.js

```diff
@@ -0,0 +1,27 @@
+let getCookie = function (cname) {
+    let name = cname + "=";
+    let decodedCookie = decodeURIComponent(document.cookie);
+    let ca = decodedCookie.split(';');
+    for (let i = 0; i < ca.length; i++) {
+        let c = ca[i];
+        while (c.charAt(0) == ' ') {
+            c = c.substring(1);
+        }
+        if (c.indexOf(name) == 0) {
+            return c.substring(name.length, c.length);
+        }
+    }
+    return "";
+};
+
+let setCookie = function (cname, cvalue, exdays) {
+    let d = new Date();
+    d.setTime(d.getTime() + (exdays * 24 * 60 * 60 * 1000));
+    let expires = "expires=" + d.toUTCString();
+    document.cookie = cname + "=" + cvalue + ";" + expires + ";path=/";
+};
+
+let switchToPageNumber = function (pageNum) {
+
+    window.location = "?page=" + pageNum;
```

Comment:
Guys, I wonder if using links on the buttons would be a cleaner approach than this? Then hovering over the button you can see the page number where the button takes you. I know Venia doesn't have this, so this is just a question.

Reply:
Links are now generated without JS functions,